ui/imports/shared/views/ProfileView.qml

```diff
@@ -0,0 +1,238 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+import QtGraphicalEffects 1.13
+
+import utils 1.0
+import shared 1.0
+import shared.popups 1.0
+import shared.stores 1.0
+import shared.controls.chat 1.0
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Popups 0.1
+
+Rectangle {
+    id: root
+
+    property Popup parentPopup
+
+    property var profileStore
+    property var contactsStore
+
+    property string userPublicKey: profileStore.pubkey
+    property string userDisplayName: profileStore.displayName
+    property string userName: profileStore.username
+    property string userNickname: ""
+    property string userEnsName: profileStore.ensName
+    property string userIcon: profileStore.profileLargeImage
+    property string text: ""
+
+    readonly property int innerMargin: 20
+
+    property bool userIsEnsVerified: false
+    property bool userIsBlocked: false
+    property bool isCurrentUser: profileStore.pubkey === userPublicKey
+    property bool isAddedContact: false
+
+    readonly property string subTitle: userIsEnsVerified ? userName : Utils.getElidedCompressedPk(userPublicKey)
```

Comment:
@igor-sirotin please consider this and some of props around... some of them may be part of a private section, we don't need to expose all of them?
```
QtObject {
    id: d
    // props...
}
```